src/modules/apiHelper.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const articlesIndex = async () => {
+  try {
+    const response = await axios.get("/api/articles/");
+    return response.data.articles;
+  } catch (error) {
+    errorHandler(error);
+  }
+};
```

Comment:
This is totally okay, but it is not what I suggested during our discussion about how to extract this functionality. If it was hard to follow my reasoning, you should ask for clarification.